pkg/validate/security_context.go

```diff
@@ -225,7 +247,7 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		})
 	})
 
-	Context("runtime should support container with security context", func() {
+	Context("bucket", func() {
```

Comment:
What do you mean by "bucket"?

Reply:
a bucket of tests...  This is the second "bucket" of bucket-2, would be another possible name.  This context is already within the "Security Context" .. the other buckets at this level are NamespaceOption and SeccompProfilePath.  

This bucket comprises tests for: SupplementalGroups, RunAsUser, RunAsUserName, ReadOnlyRootfs, Privileged, and Capability. I couldn't come up with a better name for these than the "Security Context bucket"